### DISCUSSION TITLE
更新主題到最新之後版面全部跑位

### DISCUSSION THREAD

**Discussion by Peckystudios** [CaiJimmy/hugo-theme-stack#907] [Q&A]
大家好
不好意思請問有人跟我遇到一樣的問題嗎？

更新主題之後我的部落格網站的界面都跑掉了
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/84195047/36cdbce6-74a4-424c-9e8f-655e0dad27f7)

我的github repo
https://github.com/Peckystudios/PeckyPlayZone

謝謝大家><

**Comment by chuxinyuan** [comment]
不确定是不是hugo版本的问题。

**Reply by Peckystudios** [reply]
請問是需要進行更新嗎@@？不太確定怎麼更新

**Reply by chuxinyuan** [reply]
我建议你根据配置文件的要求更新到 hugo v0.117.0 版本。统计之都上有一篇文章你可以看看。

**Reply by Peckystudios** [reply]
謝謝你回覆
我在統計之都沒有搜尋到相關文章
不過嘗試更新了hugo
但還是有一樣的問題
網站可以build但版面不正常

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/84195047/05762079-3d34-43ed-82f2-040bab212008)

**Comment by CaiJimmy** [comment]
试试看删除仓库里的 `resources` 文件夹。

**Reply by Peckystudios** [reply]
HI
不好意思今天才有空測試
我有刪除再使用hugo server
他有生成新的resource
但是打開local的那個網址
版型還是和原本一樣
不知道為什麼xd

謝謝你回覆！！
